src/app/shared/components/language-select/language-select.component.ts

```diff
@@ -6,22 +6,25 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./language-select.component.scss']
 })
 export class LanguageSelectComponent implements OnInit {
-  ru = 'ru';
-  eng = 'eng';
-  selectedRu = '';
-  selectedEng = 'selected';
+  obj: any[] = [
```

Comment:
Just a reminder to get rid of any[] one day)

Reply:
No, my comment was about to define type of the array using _type_ or _interface_, like 
```
interface LanguageConf {
  lang: string,
  selected: boolean
}

obj: LanguageConf[] =[...]
```
Leaving variables types as any is really not recommended